### PR TITLE
[EA] deploy subforum threads on frontpage, audio feature pulse, updates to deleted comments, various other things

### PR DESCRIPTION
**Pull request by s-cheng**
┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1203769731178012) by [Unito](https://www.unito.io)

### CODE REVIEW

File: packages/lesswrong/components/comments/CommentsItem/CommentsItem.tsx [hunk 1]
```diff
@@ -418,6 +419,7 @@ export const CommentsItem = ({ treeOptions, comment, nestingLevel=1, isChild, co
               <div className={classes.usernameSpacing}>○</div>
             }
             {post && <CommentShortformIcon comment={comment} post={post} />}
+            {displayTagIcon && <span>TODO</span>}
```
Comment (s-cheng) [inline packages/lesswrong/components/comments/CommentsItem/CommentsItem.tsx:422]: @jpaddison3 Is this related to https://github.com/ForumMagnum/ForumMagnum/pull/6456? Did you mean to fill this in?

File: packages/lesswrong/lib/collections/comments/fragments.ts [hunk 2]
```diff
@@ -136,6 +139,8 @@ registerFragment(`
   }
 `);
 
+// TODO: This is now the same as CommentWithRepliesFragment, now that said
+// fragment gets the tag field
 registerFragment(`
   fragment StickySubforumCommentFragment on Comment {
     ...CommentWithRepliesFragment
```
Comment (s-cheng) [inline packages/lesswrong/lib/collections/comments/fragments.ts:146]: @jpaddison3 Is this a TODO you intended to address in your PR?

Comment (jpaddison3) [reply packages/lesswrong/lib/collections/comments/fragments.ts:146]: Nope

Comment (jpaddison3) [reply packages/lesswrong/lib/collections/comments/fragments.ts:146]: (I didn't introduce the reproduction)

File: packages/lesswrong/components/comments/CommentsItem/CommentsItem.tsx (hunk 1 above)
Comment (jpaddison3) [reply packages/lesswrong/components/comments/CommentsItem/CommentsItem.tsx:422]: Yikes

**Pull request merged**